### ISSUE TITLE
remove libavl dependency

### ISSUE THREAD

**Issue by munkyboy** [jasonmc/forked-daapd#14]
is it possible to replace the libavl dependency with something that's actively maintained and/or tested on multiple OSes? I'm having a tough time getting it installed on FreeBSD 8.x AMD64...

**Comment by ghost** [comment]
I agree, the major problems on FreeBSD are the unmaintained version of libavl and antlr3.  If these dependencies could be removed, it would make FreeBSD compilation easier.

**Comment by tonyklawrence** [comment]
I'd love to build this on my ReadyNAS which is a v.old debian.
Removing these would be great.
